packages/expo-secure-store/android/src/main/java/expo/modules/securestore/AuthenticationCallback.kt

```diff
@@ -0,0 +1,28 @@
+package expo.modules.securestore
+
+import expo.modules.core.Promise
+import expo.modules.core.arguments.ReadableArguments
+import javax.crypto.Cipher
+import javax.crypto.spec.GCMParameterSpec
+
+// Interface used to pass the authentication logic
+interface AuthenticationCallback {
+  fun checkAuthentication(
+      promise: Promise,
+      cipher: Cipher,
+      gcmParameterSpec: GCMParameterSpec,
+      options: ReadableArguments,
+      encryptionCallback: EncryptionCallback,
+      postEncryptionCallback: PostEncryptionCallback?
+  )
+
+  fun checkAuthentication(
+      promise: Promise,
+      requiresAuthentication: Boolean,
+      cipher: Cipher,
+      gcmParameterSpec: GCMParameterSpec,
+      options: ReadableArguments,
+      encryptionCallback: EncryptionCallback,
+      postEncryptionCallback: PostEncryptionCallback?
+  )
+}
```

Comment:
```suggestion
}

```
We use newlines at the of files. Not having them causes CI lint to fail.

_Hint: You can run `./gradlew :expo-secure-store:spotlessApply` to auto-format Kotlin code 😉 _

Reply:
Thanks for the tip, otherwise I would have been changing it by hand 😄